gallery/src/pages/Accessibility.tsx

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import {GalleryTile} from '../gallery-components/GalleryTiles';
+import { NxP, NxCode, NxH3, NxList, NxTextLink } from '@sonatype/react-shared-components';
+
+const Accessibility = () =>
+  <GalleryTile title="Accessibility">
+    <NxP>
+      Accessibility (A11y) means ensuring that our applications are usable by everybody. We do this in a variety of
+      different ways. We make sure that our designs are clear and that they follow accessibility guidelines for
+      contrast and readability. Our components are written with semantic HTML, correct use of <NxCode>aria</NxCode>
+      tags, and testing in VoiceOver and ChromeVox accessibility is built into the Shared Components. However there is
+      still work that needs to be done to make sure that the application as a whole is accessible.
+    </NxP>
+
+    <NxH3 className="nx-tile__section-header">Making your pages accessible</NxH3>
+    <NxP>
+      Ensuring you use the components in an accessible manner is important, here are a few tips to help you make your
+      pages accessible.
+    </NxP>
+
+    <NxList className="nx-list--bulleted">
+      <NxList.Item>Tab order is important. Make sure that the tab order of the page is logical.</NxList.Item>
+      <NxList.Item>Related to the above, does the keyboard focus order follow the visual layout?</NxList.Item>
+      <NxList.Item>Unique title element for each page or view?</NxList.Item>
+      <NxList.Item>Do all images and visual graphs have alt text?</NxList.Item>
+      <NxList.Item>
+        Make sure that you take advantage of the props and attributes that have been added to the components to improve
+        their accessibility. Many component documentation pages have an Accessibility section with extra information.
+      </NxList.Item>
+      <NxList.Item>
+        <NxTextLink external href="https://developer.mozilla.org/en-US/docs/Glossary/Semantics">
+          Semantic HTML
+        </NxTextLink>
+        {' '}is a critical aspect of accessibility. Headings should always use the correct
+        {' '}<NxCode>&lt;h#&gt;</NxCode> tag, paragraphs should be wrapped in <NxCode>&lt;p&gt;</NxCode> tags, etc.
+      </NxList.Item>
```

Comment:
Also recommend the use of `<section>` and mention which RSC components generate sections

Reply:
> Also recommend the use of `<section>` and mention which RSC components generate sections

For the Accessibility page I had avoided mentioning specific components and what they might or might not do because I didn’t want us to be constantly updating this page when we change/add a component. To me the accessibility of specific components should be described on their pages, this is more general guidance and best practices.

Reply:
> Also recommend the use of `<section>` and mention which RSC components generate sections

Also revised the Semantic HTML a bit here: https://github.com/sonatype/sonatype-react-shared-components/pull/455/commits/e9cdbae359e7c142327be871d1a4c5a4df5aba40

Reply:
> Also revised the Semantic HTML a bit here: e9cdbae

I'm not sure how I feel about the `#react-components` reference in what is technically public documentation.

Reply:
> > Also revised the Semantic HTML a bit here: [e9cdbae](https://github.com/sonatype/sonatype-react-shared-components/commit/e9cdbae359e7c142327be871d1a4c5a4df5aba40)
> 
> I'm not sure how I feel about the `#react-components` reference in what is technically public documentation.

I created https://issues.sonatype.org/browse/RSC-655 so that we can discuss because references to #react-components appear on two other RSC pages as well